database/migrations/2021_02_25_175500_create_booking_status_table.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateBookingStatusTable extends Migration
+{
+    public function up()
+    {
+        Schema::create('booking_status', function (Blueprint $table) {
+            $table->id();
+            $table->string('label');
+            $table->timestamps();
+        });
+    }
+}
```

Comment:
@sl0wik Why can't we use the tipoff/statuses package for this?

Reply:
It was still in an attempt to make it a clean package :). We can use tipoff/statuses if you prefer. Having a booking status separate example purpose: if one day there is a feature request where some status can be set only by admin (like refunded), then status cant be extended for that purposes.

Reply:
Understood. I think it is worth using that package and if features like that need to be added to the other package, we can make the changes there. Statuses was originally created to be added to bookings and orders.

Reply:
I created issue #41 for this